src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -285,6 +290,14 @@ public int getPendingCompactions() {
    * @return true if any repairs are running on the node.
    */
   public boolean isRepairRunning() {
+    return isRepairRunningPre22() || isRepairRunningPost22() || isValidationCompactionRunning();
```

Comment:
Do we need here `isAntiEntropyStage()` as well? From [nodetool tpstats](http://docs.datastax.com/en/cassandra/3.0/cassandra/tools/toolsTPstats.html) documentation it says it handles repair messages (merkle tree requests etc.) which can indicate that repair is running as well.